librtasevent_src/rtas_srcfru.c

```diff
@@ -73,13 +73,16 @@ parse_fru_id_scn(struct rtas_event *re)
     parse_fru_hdr(&fru_id->fruhdr, &fru_id_raw->fruhdr);
     re->offset += RE_FRU_HDR_SZ;
 
+    int len_procedure_id = strlen(fru_id->procedure_id);
+    int len_part_no = strlen(fru_id->part_no);
+
```

Comment:
You have to declare variable at the begining of fuction/code block. Else compiler will complain.
Also above one won't work.